zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/integration/DownloadUtils.java

```diff
@@ -80,9 +80,6 @@ public static String downloadFlink(String flinkVersion, String scalaVersion) {
       runShellCommand(new String[]{"wget",
               "https://repo1.maven.org/maven2/org/apache/hive/hive-exec/2.3.4/hive-exec-2.3.4.jar",
               "-P", targetFlinkHomeFolder + "/lib"});
-      runShellCommand(new String[]{"wget",
-              "https://repo1.maven.org/maven2/org/apache/flink/flink-shaded-hadoop2-uber/2.7.5-1.8.1/flink-shaded-hadoop2-uber-2.7.5-1.8.1.jar",
```

Comment:
`hadoop-client-api` and `hadoop-client-runtime` should be used instead, see 
- https://issues.apache.org/jira/browse/FLINK-11086
- https://www.mail-archive.com/dev@flink.apache.org/msg37323.html
- https://github.com/apache/flink/pull/11983